src/client/components/SocialLink.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SocialLink = ({ profile, url }) => (
+  <div>
+    <i
+      className={`iconfont text-icon icon-${profile.icon}`}
+      style={{
+        color: profile.color,
+      }}
+    />
+    <a target="_blank" rel="noopener noreferrer" href={url}>
+      {profile.name}
+    </a>
+  </div>
+);
+
+SocialLink.propTypes = {
+  profile: PropTypes.shape({
+    id: PropTypes.string.isRequired,
```

Comment:
should we start defining the properties for certain shapes now for propTypes?

Reply:
If we know shape ahead of time, and it doesn't require much work to define properties it might be useful.